services/order.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+from django.shortcuts import get_object_or_404
+
+from db.models import Order, Ticket
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    user = get_object_or_404(get_user_model(), username=username)
+
+    order = Order.objects.create(
+        user=user
+    )
+
+    if date:
+        order.created_at = date
+        order.save()
+
+    for ticket in tickets:
+        Ticket.objects.create(
+            row=ticket.get("row"),
+            seat=ticket.get("seat"),
+            movie_session_id=ticket.get("movie_session"),
+            order=order
+        )
```

Comment:
The `create_order` function should validate the ticket data to ensure that the `row`, `seat`, and `movie_session` fields are valid. Consider using `get_object_or_404` to validate the `movie_session` ID before creating the ticket.